libretroshare/src/services/autoproxy/rsautoproxymonitor.cc

```diff
@@ -168,7 +174,11 @@ void rsAutoProxyMonitor::task(taskTicket *ticket)
 			*tt = *ticket;
 			tt->types.clear();
 			tt->types.push_back(*it);
-			s->processTaskAsync(tt);
+
+			// it's async!
+			RsThread::async([s, tt] {
+				s->processTaskAsync(tt);
```

Comment:
Maybe it is better to put `RsThread::async` directly inside `processTaskAsync` so the name of the function is not misleading anymore.